UnityProject/Assets/Editor/LevelBuilder/Util/GUIUtils.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+public class GUIUtils
+{
+	// A utility method that takes creates EditorGUILayout.TextField
+	// with no space between the label and the textfield.
+	public static string TextField(string label, string text)
+	{
+		var textDimensions = GUI.skin.label.CalcSize (new GUIContent (label));
+		EditorGUIUtility.labelWidth = textDimensions.x;
+
+		return EditorGUILayout.TextField (label, text);
+	}
+}
```

Comment:
new line please? @ramonfmir might notice :D

Reply:
Realised it was deleted after my commit - just fixed and rebased them! :D